packages/samples/atomic-react/src/HeaderLink.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {FunctionComponent} from 'react';
+
+type Props = {
+  page: string;
+  currentPage: string;
+  setPage(page: string): void;
+};
+export const HeaderLink: FunctionComponent<Props> = ({
```

Comment:
I would nest this component in a directory like `src/utils` to avoid too much clutter at the base `src` directory, where developers may first look.